pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -0,0 +1,38 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore": "5.0.1-rc3-24126-00",
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.1-rc3-24126-00",
+    "Microsoft.NETCore.Runtime": "1.0.2-rc3-24126-00",
```

Comment:
This is not new the UWP  meta-package has always pulled in the CoreCLR package. It will pull whichever version of the package we are actually publishing, whether that comes from open or closed doesn't matter it just gets it from the feed.